<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>665</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>476</width>
    <height>665</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>476</width>
    <height>665</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>主界面</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/image/icon.jpg</normaloff>:/image/icon.jpg</iconset>
  </property>
  <widget class="QPushButton" name="pb_uploadc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>上传续传</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_downloadc">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下载续传</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_num">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>95</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pb_upload">
      <property name="text">
       <string>上传</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_download">
      <property name="text">
       <string>下载</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pb_delete">
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tb_V1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>341</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTableView" name="tb_V2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </widget>
  <widget class="QTableView" name="tb_V3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>550</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lb_filenum">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>355</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string> 已上传文件数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
